internal/dataexchange/ffdx/dxevent.go

```diff
@@ -170,25 +183,16 @@ func (h *FFDX) dispatchEvent(msg *wsEvent) {
 				PayloadRef: msg.Path,
 			}
 		}
-	case blobAcknowledged:
-		h.callbacks.OperationUpdate(h.ctx, &core.OperationUpdate{
-			NamespacedOpID: msg.RequestID,
-			Status:         core.OpStatusSucceeded,
-			Output:         msg.Info,
-			VerifyManifest: h.capabilities.Manifest,
-			DXHash:         msg.Hash,
-			OnComplete:     e.Ack,
-		})
-		return
+
 	default:
 		err = i18n.NewError(h.ctx, coremsgs.MsgUnexpectedDXMessageType, msg.Type)
 	}
 
 	// If we couldn't dispatch the event we received, we still ack it
-	if err == nil {
-		h.callbacks.DXEvent(h.ctx, namespace, e)
-	} else {
+	if err != nil {
 		log.L(h.ctx).Warnf("Failed to dispatch DX event: %s", err)
 		e.Ack()
+	} else {
+		h.callbacks.DXEvent(h.ctx, namespace, msg.Recipient, e)
```

Comment:
Can we always assume `Recipient` will be set now, or do we need a fallback if it's not?

Reply:
Yes, there recipient should always be present in the blob received events.

Reply:
Confirmed with @gabriel-indik - yes it's always set.